Lib/ensurepip/__init__.py

```diff
@@ -14,39 +16,51 @@
 # policies recommend against bundling dependencies. For example, Fedora
 # installs wheel packages in the /usr/share/python-wheels/ directory and don't
 # install the ensurepip._bundled package.
-_WHEEL_PKG_DIR = sysconfig.get_config_var('WHEEL_PKG_DIR')
+_WHEEL_PKG_DIR = (
+    whl_pkg_dir_str := sysconfig.get_config_var('WHEEL_PKG_DIR')
+) is not None and Path(whl_pkg_dir_str).resolve() or None
```

Comment:
This is fairly difficult to parse and understand correctly. There's no real strong reason to make this a single expression.

Here's a suggestion I have:

```suggestion
if (_pkg_dir := sysconfig.get_config_var('WHEEL_PKG_DIR')) is not None:
    _WHEEL_PKG_DIR = Path(_pkg_dir).resolve()
else:
    _WHEEL_PKG_DIR = None
```